var/spack/repos/builtin/packages/py-mpi4py/package.py

```diff
@@ -35,8 +35,3 @@ class PyMpi4py(PythonPackage):
     @when('@3.1:')
     def install_options(self, spec, prefix):
         return ['--mpicc=%s -shared' % spec['mpi'].mpicc]
-
-    @property
-    def headers(self):
-        headers = find_all_headers(self.prefix.lib)
-        return headers
```

Comment:
I'm confused: I assume `prefix.lib` and `platlib` are separate directories. https://github.com/spack/spack/pull/17295 explicitly added this, which I assume means that header files are in fact included there (or did that change when the installation mechanism for most `py-*` packages changed?).

Reply:
`platlib` is a specific sub-directory of `prefix.lib`. For example, `lib/pythonX.Y/site-packages` (or `lib64/...` on RHEL).

Reply:
I forgot that `find_all_headers` searched recursively, so I'm guessing that this can be removed since starting the search from `platlib` has the same result.